src/scripts/i18n/inline_and_diff.py

```
@@ -0,0 +1,201 @@
+# Read original files
+original_files_map = {file_path: Path(file_path).read_text(encoding='utf-8') for file_path in all_files}
+
+# ---------------------- 2. Inline translation  --------------------- #
+# Load translation JSON (en.json)
+en_json_path = Path('src/assets/i18n/en.json')
+if not en_json_path.exists():
+    sys.exit(f"Translation file not found: {en_json_path}")
+with open(en_json_path, 'r', encoding='utf-8') as f:
+    translations: Dict = json.load(f)
+
+def get_translation(key: str, tree: Dict):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_translation`

(ANN201)